<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeConvert</class>
 <widget class="QWidget" name="EncodeConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncodeConvert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/img/main.png</normaloff>:/Resources/img/main.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>filePath</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>file size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>file code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>convert code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>convert result</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>convert options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>convert to code</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="codeToComboBox">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>UTF8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UTF8 BOM</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UTF16-LE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UTF16-BE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GBK</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>deal file ext</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="extComboBox">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>all support file ext</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="toolTip">
               <string>user defined</string>
              </property>
              <property name="text">
               <string>defined</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectFileBt">
           <property name="text">
            <string>select dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startBt">
           <property name="text">
            <string>start</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeBt">
           <property name="text">
            <string>close</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="logTextBrowser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>selectFileBt</sender>
   <signal>clicked()</signal>
   <receiver>EncodeConvert</receiver>
   <slot>slot_selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startBt</sender>
   <signal>clicked()</signal>
   <receiver>EncodeConvert</receiver>
   <slot>slot_startConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBt</sender>
   <signal>clicked()</signal>
   <receiver>EncodeConvert</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>EncodeConvert</receiver>
   <slot>slot_userDefineExt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_selectFile()</slot>
  <slot>slot_startConvert()</slot>
  <slot>slot_userDefineExt()</slot>
 </slots>
</ui>
